CONTRIBUTING.md

```diff
@@ -85,25 +85,28 @@ development with the following:
 
   For Url:
 
-  `http://localhost:5000/`
+  `http://localhost:5000/auth/myusa/callback`
 
   For Redirect uri:
 
   `http://localhost:5000/auth/myusa/callback`
 
-2. Generate a set of keys, `MYUSA_KEY` and `MYUSA_SECRET`, and reference them in
+2. Under `Select the API Scopes that your Application will use`: select `Email
+   Address`.
+
+3. Generate a set of keys, `MYUSA_KEY` and `MYUSA_SECRET`, and reference them in
```

Comment:
Hmm good question. @afeld @wslack @NoahKunin do any of you know if it is possible to have a public slack environment for guests who are interested in using and contributing to our bots?